daemon/api.go

```diff
@@ -2825,3 +2841,60 @@ func postApps(c *Command, r *http.Request, user *auth.UserState) Response {
 	st.EnsureBefore(0)
 	return AsyncResponse(nil, &Meta{Change: chg.ID()})
 }
+
+var (
+	stateOkayWarnings   = (*state.State).OkayWarnings
+	stateAllWarnings    = (*state.State).AllWarnings
+	stateWarningsToShow = (*state.State).WarningsToShow
```

Comment:
The ack verb also used here feels a bit more clear than "Okay":

How about:

* AckWarnings
* AllWarnings
* PendingWarnings

Reply:
the user command would still be `okay`?